.vscode/launch.json

```diff
@@ -1,11 +1,62 @@
 {
   "version": "0.2.0",
   "configurations": [
-      {
-          "type": "pwa-node",
-          "request": "attach",
-          "name": "attach-to-service",
-          "port": 9229
-      }
+    /**
+     * Attaches a debugger to an already running OSD instance
+     */
+    {
+      "name": "Attach Debugger to running Dashboards Server",
+      "type": "pwa-node",
+      "request": "attach",
+      "port": 9229
+    },
+    /**
+     * Starts the OSD instance only
+     */
+    {
+      "name": "Start Dashboards Server",
```

Comment:
this should just be like `Start Dashboards`

the config above will attach directly to the Node server (which includes the code in the server folders).

But this will start OSD completely so it conflicts with the first config title

Reply:
Noted. I guess from my perspective, `Start Dashboards` sounds kinda confusing since I'd expect everything to just "work", which is why I added the "Server" keyword in the launch name